src/plugins/sorting-task/SortingTaskNode/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import EditView from "./EditView";
+import ReadView from "./ReadView";
+
+import { ITermsList } from "./interfaces";
+
+interface IProps {
+  attributes: object,
+  children: [ React.Component ],
+  readOnly: boolean,
+  editor: any,
```

Comment:
Clarfiy that `editor`, `node`, `parent` are set by Slate. Might be sensible to add Slate-specific type defs.